### PR TITLE
Yohannes week 2

### PR DESCRIPTION
**Pull request by byohannes**

### CODE REVIEW

File: week-2/Homework/extra/4-radio-stations.js [hunk 1]
```diff
@@ -25,57 +34,78 @@
...
+    this.stations = availableStations
   }
 
-  return this.stations.includes(frequency);
+  return this.stations.includes(frequency)
 }
 
-const assert = require("assert");
-
+const assert = require('assert')
```
Comment (sansaid) [inline week-2/Homework/extra/4-radio-stations.js]: I'm really glad to see you experimenting with the `require` statement here! Did you put this in yourself @byohannes ?

File: week-2/Homework/mandatory/2-function-creation.js [hunk 2]
```diff
@@ -5,7 +5,20 @@ Write a function that:
...
+  for (let i = 0; i < strArr.length; ++i) {
+    /*replacing any forward slashes (/) in the strings 
+    and replace it with an empty space 
+    and then trimming the paces and 
+    after changing all to lower case.
+    */
+    strArr[i] = strArr[i]
+      .replace('/', '')
+      .trim()
+      .toLowerCase()
```
Comment (sansaid) [inline week-2/Homework/mandatory/2-function-creation.js]: Nice! Good job chaining the methods! 👍

**Comment by sansaid** [comment]
Great job @byohannes ! Found it difficult to find any issues with your code - spot on! :)

File: week-2/Homework/extra/4-radio-stations.js (hunk 1 above)
Comment (byohannes) [reply week-2/Homework/extra/4-radio-stations.js]: @sansaid it was given readily within the test code but I have researched about what it does as // Generate an AssertionError to compare the error message

**Pull request closed without merging**